main.py

```diff
@@ -1,14 +1,16 @@
 import json
 import math
+from operator import inv
 from base import Base
```

Comment:
I would choose a different variable name for this, there are a lot of `base` or `Base` used already

Reply:
Gotcha.